Samples/README.md

```diff
@@ -1,13 +1,131 @@
+    });
+}
+
+app.get('/', async (req, res) => {
+  let bearerToken = await getSkyflowBearerToken();
+  res.json({"accessToken" : bearerToken});
+})
+
+app.listen(port, () => {
+  console.log(`Server is listening on port ${port}`)
+})
+```
+8. Start the server
+
+        node index.js
+    server will start at `localhost:3000`
+9. Your **<TOKEN_END_POINT_URL>** with `http://localhost:3000/`
+
+## The samples
+### Collect and reveal
+This sample illustrates how to use secure Skyflow elements to collect sensitive user information and reveal it to the user via tokens.
+#### Configure
+1. In `Skyflow.Configuration()` of [ViewController.swift](CollectAndRevealSample/CollectAndRevealSample/ViewController.swift), replace with the following fields:
+         - Replace the placeholder "<VAULT_ID>" with the correct vaultId you want to connect
+         - Replace the placeholder "<VAULT_URL>" with the correct vaultURL
+2. Update `Fields` struct in [ResponseStructs.swift](CollectAndRevealSample/CollectAndRevealSample/ResponseStructs.swift) with field name of used vault. For ex: 
+        
+        ```swift
+            struct Fields: Codable {
+               let name: NameField
+               let cvv: String
+               let cardExpiration: String
+               let cardNumber: String
+               let skyflow_id: String
+            }
+        ```
+        
+    The fields can be different depending upon the vault.
+3. Update 
+4. Replace the placeholder "<TOKEN_END_POINT_URL>" of [ExampleTokenProvider.swift](CollectAndRevealSample/CollectAndRevealSample/ExampleTokenProvider.swift) with the  bearer token endpoint which gives the bearerToken, implemented at your backend or `http://localhost:3000/`.
+ 5. Running the sample
+    1. Open CMD
+    2. Navigate to `CollectAndRevealSample`
+    3. Run 
+
+            pod install
+    4. Open the [`CollectAndRevealSample.xcworkspace](`CollectAndRevealSample/`CollectAndRevealSample.xcworkspace) file using xcode
+    5. click on build and run
+
+### Validations
+This sample illustrates how to apply custom validation rules on secure Skyflow Collect elements to restrict the type of input a user can provide.
+#### Configure
+1. In `Skyflow.Configuration()` of [ViewController.swift](Validations/Validations/ViewController.swift), replace with the following fields:
+2. Replace the placeholder "<VAULT_ID>" in the configuration with the correct vaultId you want to connect
+3. Replace the placeholder "<VAULT_URL>" with the correct vaultURL
+4. Replace the placeholder "<TOKEN_END_POINT_URL>" in [ExampleTokenProvider.swift](Validations/Validations/ExampleTokenProvider.swift) with the  bearer token endpoint which gives the bearerToken, implemented at your backend or `http://localhost:3000/`.
+5. Running the sample
+      1. Open CMD
+      2. Navigate to `Validations`
+      3. Run 
+            
+             pod install
+      4. Open the [Validations.xcworkspace](Validations/Validations.xcworkspace) file using xcode
```

Comment:
This file doesn't exist.

Reply:
Validations.xcworkspace does not exists in github repo. It automatically comes when we clone the repo in our local.